### ISSUE TITLE
Support for credentials in JMX 

### ISSUE THREAD

**Issue by blackside** [threelegs/newrelic-plugins#20]
I don't see any support for passing along JMX credentials. Is it possible to add support for that?

**Comment by cjlogan1721** [comment]
Yes, this is very important for me and a show-stopper.  Not only that, it's very easy to add.

**Comment by fchambo** [comment]
+1

**Comment by thoersch** [comment]
We're currently supporting cassandra 2.x+ and JMX authentication: https://github.com/thoersch/new-relic-cassandra
